blocks/HARDWARE/PROTOCOLS/CAN/UTILS/DECODE_CAN_MESSAGE/DECODE_CAN_MESSAGE.py

```diff
@@ -0,0 +1,46 @@
+from flojoy import flojoy, Stateful, DataFrame
+import pandas as pd
+import logging
+
+
+@flojoy(deps={"python-can": "4.3.1", "cantools": "39.4.2"})
+def DECODE_CAN_MESSAGE(
+    dbc: Stateful,
+    messages: Stateful,
+    ignore_undefined_id_error: bool = False,
+) -> DataFrame:
+    """DECODE_CAN a CAN message.
+
+    DECODE_CAN a CAN message using the provided databse.
+
+    Parameters
+    ----------
+    dbc : Stateful
+        The database to use for decoding the message.
+    messages : Stateful
+        The message to DECODE_CAN. Must be a can.Message object.
+    ignore_undefined_id_error : bool
+        If True, ignore undefined id error. Default is False.
+
+    Returns
+    -------
+    DataFrame : DataFrame
+        Return dataframe containing the DECODE_CANd message.
```

Comment:
typo?

Reply:
Ahahah, yes, when I switched from DECODE to DECODE_CAN, I forgot to include the case-sensitive option